llpc/test/shaderdb/general/TestWorkgroupIdOpt.comp

```diff
@@ -21,7 +21,7 @@ void main()
 // CHECK-NEXT:    [[TMP0:%.*]] = call i64 @llvm.amdgcn.s.getpc()
 // CHECK-NEXT:    [[TMP1:%.*]] = and i64 [[TMP0]], -4294967296
 // CHECK-NEXT:    [[TMP2:%.*]] = zext i32 [[USERDATA0]] to i64
-// CHECK-NEXT:    [[TMP3:%.*]] = or i64 [[TMP1]], [[TMP2]]
+// CHECK-NEXT:    [[TMP3:%.*]] = or {{(disjoint )?}}i64 [[TMP1]], [[TMP2]]
```

Comment:
This test has auto-generated checks - presumably the optional disjoint will get changed if that's run?

Reply:
Yeah, once llvm is updated and someone regenerates the test, these will be made permanent :)